edge/pkg/metamanager/apiserver-lite/server.go

```diff
@@ -0,0 +1,169 @@
+package apiserver_lite
+
+import (
+	"context"
+	_ "context"
+	"fmt"
+	"github.com/kubeedge/kubeedge/edge/pkg/metamanager/apiserver-lite/handlerfactory"
+	"github.com/kubeedge/kubeedge/edge/pkg/metamanager/apiserver-lite/kubernetes/serializer"
+	"github.com/kubeedge/kubeedge/edge/pkg/metamanager/apiserver-lite/kubernetes/storage/cacher"
+	"github.com/kubeedge/kubeedge/edge/pkg/metamanager/apiserver-lite/kubernetes/storage/sqlite"
+	"github.com/kubeedge/kubeedge/edge/pkg/metamanager/apiserver-lite/kubernetes/storage/sqlite/imitator"
+	"github.com/kubeedge/kubeedge/pkg/apiserverlite"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/sets"
+	utilwaitgroup "k8s.io/apimachinery/pkg/util/waitgroup"
+	genericapifilters "k8s.io/apiserver/pkg/endpoints/filters"
+	"k8s.io/apiserver/pkg/endpoints/handlers/negotiation"
+	"k8s.io/apiserver/pkg/endpoints/handlers/responsewriters"
+	apirequest "k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/apiserver/pkg/server"
+	genericfilters "k8s.io/apiserver/pkg/server/filters"
+	"k8s.io/apiserver/pkg/storage"
+	"k8s.io/klog/v2"
+	"net/http"
+	"time"
+)
+
+const(
+	httpaddr = "0.0.0.0:10010"
```

Comment:
Is it necessary for all or just listen on 127.0.0.1?

Also add TODO for addr config.